From c5f4dcb3bdd393992f665a92f930e68e4f5d7005 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:39:28 -0400
Subject: [alsa] hda: Add mute LED pin quirk for HP 15 touchsmart

Message-id: <140972823573-397-patchset-jkysela@redhat.com>
Patchwork-id: 91093
O-Subject: [RHEL 7.1 PATCH 396/442] ALSA: hda - Add mute LED pin quirk for HP 15 touchsmart
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

This makes the mute LED work on a HP 15 touchsmart machine.

BugLink: https://bugs.launchpad.net/bugs/1334950
Signed-off-by: David Henningsson <david.henningsson@canonical.com>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: 423044744aa4c250058e976474856a7a41972182)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_realtek.c b/sound/pci/hda/patch_realtek.c
index 512582b..6f204ba 100644
--- a/sound/pci/hda/patch_realtek.c
+++ b/sound/pci/hda/patch_realtek.c
@@ -5160,6 +5160,17 @@ static const struct snd_hda_pin_quirk alc269_pin_fixup_tbl[] = {
   {0x1d, 0x40700001},
   {0x1e, 0x411111f0},
   {0x21, 0x02211040}),
+ SND_HDA_PIN_QUIRK(0x10ec0282, 0x103c, "HP 15 Touchsmart", ALC269_FIXUP_HP_MUTE_LED_MIC1,
+  {0x12, 0x99a30130},
+  {0x14, 0x90170110},
+  {0x17, 0x40000000},
+  {0x18, 0x411111f0},
+  {0x19, 0x03a11020},
+  {0x1a, 0x411111f0},
+  {0x1b, 0x411111f0},
+  {0x1d, 0x40f41905},
+  {0x1e, 0x411111f0},
+  {0x21, 0x0321101f}),
  SND_HDA_PIN_QUIRK(0x10ec0283, 0x1028, "Dell", ALC269_FIXUP_DELL1_MIC_NO_PRESENCE,
   {0x12, 0x90a60130},
   {0x14, 0x90170110},
-- 
1.7.1